src/include/binder/binder.h

```diff
@@ -37,36 +37,64 @@
 #include <string>
 
 #include "binder/simplified_token.h"
+#include "binder/statement/create_statement.h"
 #include "catalog/column.h"
+#include "common/macros.h"
 #include "nodes/parsenodes.hpp"
 #include "type/type_id.h"
 #include "type/value.h"
 
+#include "binder/binder.h"
```

Comment:
Maybe I'm not seeing this correctly, we are including ourself?

Most of these includes could probably be fwd decls, granted I think there shouldn't be too many things (transitively) importing binder.h

Reply:
You're write, I'm surprised it's not detected!

I'll do forward decl.